inc/views/base/postbit/postbit.twig

```diff
@@ -0,0 +1,46 @@
+{{ ignore_bit|raw }}{{ deleted_bit|raw }}
+<a name="pid{{ post.pid }}" id="pid{{ post.pid }}"></a>
+<div class="post {{ unapproved_shade }}" style="{{ post_visibility }}" id="post_{{ post.pid }}">
+	<div class="post_author">
+		{{ post.useravatar|raw }}
+		<div class="author_information">
+				<strong><span class="largetext">{{ post.profilelink|raw }}</span></strong> {{ post.onlinestatus|raw }}<br />
+				<span class="smalltext">
+					{{ post.usertitle|raw }}<br />
+					{{ post.userstars|raw }}
+					{{ post.groupimage|raw }}
+				</span>
+		</div>
+		<div class="author_statistics">
+			{{ post.user_details|raw }}
+		</div>
+	</div>
+	<div class="post_content">
+		<div class="post_head">
+			{{ post.posturl|raw }}
+			{{ post.icon|raw }}
+			<span class="post_date">{{ post.postdate|raw }} <span class="post_edit" id="edited_by_{{ post.pid }}">{{ post.editedmsg|raw }}</span></span>
+			{{ post.subject_extra|raw }}
+		</div>
+		<div class="post_body scaleimages" id="pid_{{ post.pid }}">
+			{{ post.message|raw }}
+		</div>
+		{{ post.attachments|raw }}
+		{{ post.signature|raw }}
+		<div class="post_meta" id="post_meta_{{ post.pid }}">
+			{{ post.iplogged|raw }}
+			<div class="float_right">
+				{{ post.poststatus|raw }}
+			</div>
+		</div>
+		{{ post.input_editreason|raw }}
+	</div>
+	<div class="post_controls">
+		<div class="postbit_buttons author_buttons float_left">
+			{{ post.button_email|raw }}{{ post.button_pm|raw }}{{ post.button_www|raw }}{{ post.button_find|raw }}{{ post.button_rep|raw }}
```

Comment:
Ouch, lots of raws here. We should probably do the conditions in the template and include other templates for each button.

Reply:
I don't think there's even a good reason to break out the buttons into separate templates, is there? They will only be used in this template when the postbits are combined.

Having the conditions in the template would be ideal.

Reply:
This is a WIP. I just did that to get me up and displaying correctly on the new template. The next move is to walk back through the included templates and work them out one at a time.

Reply:
Justin ninja'd me. I was thinking of passing an array of boolean values to Twig and just `if` each button. What do you guys think?

Reply:
That sounds like the best approach to me @WildcardSearch!

Reply:
Yeah, @euantorano, that sounds like a plan.

Trust me, everything I do, I am thinking about how I am going to manipulate these templates in my plugins lol